spicedb/start-postgresql.sh

```diff
@@ -0,0 +1,15 @@
+set -e
+# Function to check if a command is available
+source ./spicedb/check_docker_podman.sh
+
+$DOCKER network create spicedb-net || true
```

Comment:
Not sure why we would want another way of starting postgres. We already have docker-compose (and even kind).

Reply:
I will remove it